streambed-confidant/src/lib.rs

```diff
@@ -183,12 +260,94 @@ impl SecretStore for FileSecretStore {
             Err(_) => Err(Error::Unauthorized),
```

Comment:
Can we log the error here? This is a bit more difficult to debug otherwise

Reply:
I think there's a general requirement to log errors. Can we do as a separate PR?

Reply:
Sounds good